orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/pipeline/ManualJudgmentStage.groovy

```diff
@@ -202,6 +154,7 @@ class ManualJudgmentStage implements StageDefinitionBuilder, AuthenticatedStage
   static class StageData {
     String judgmentStatus = ""
     List<Notification> notifications = []
+    List<String> selectedStageRoles = []
```

Comment:
@sanopsmx Did the UI side of this go out already? Curious if we can change `selectedStageRoles` to something a little clearer. I was using `requiredJudgmentRoles` in `ManualJudgmentAuthorization`.

Reply:
UI code was merged